src/main/java/org/project36/qualopt/config/WebConfigurer.java

```diff
@@ -11,8 +11,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.context.embedded.*;
-import org.springframework.boot.context.embedded.undertow.UndertowEmbeddedServletContainerFactory;
+//import org.springframework.boot.web.embedded.*;
```

Comment:
super nit: maybe remove these commented out imports/annotations in other files if they're not needed? Definitely not a big deal though - would rather have this merge and work for everyone and remove some of these later.

Reply:
Yeah forgot to remove that line. Removed it now thanks.